examples/platform/linux/AppMain.h

```diff
@@ -103,3 +103,6 @@ CommissionerDiscoveryController * GetCommissionerDiscoveryController();
 
 // For extra init calls, the function will be called right before running Matter main loop.
 void ApplicationInit();
+
+// For extra teardown calls, the function will be called right after running Matter main loop.
+void ApplicationExit();
```

Comment:
This should be called ApplicationShutdown, to make it clear that it's a parallel to ApplicationInit.  And document, perhaps, that it will be called before any of the core Matter objects are shut down?

Reply:
Done.